std/json.d

```diff
@@ -1706,12 +1706,14 @@ pure nothrow @safe unittest // issue 15884
 
 @safe unittest // issue 15885
 {
+    enum bool realInDoublePrecision = real.mant_dig == double.mant_dig;
+
     static bool test(const double num0)
     {
         import std.math : feqrel;
         const json0 = JSONValue(num0);
         const num1 = to!double(toJSON(json0));
-        version(Win32)
+        static if (realInDoublePrecision)
```

Comment:
By doing this you're relaxing the test on platforms where it currently works, inviting regressions.